azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -1384,4 +1665,161 @@ private void cleanExecutionLogs() {
           - executionLogsRetentionMs);
     }
   }
-}
+
+  /*
+   * This thread is responsible for processing queued flows.
```

Comment:
Anything special that should be called out to give readers a sense of the responsibility of this class?